components/form/FormItem/index.tsx

```diff
@@ -35,6 +35,12 @@ type RenderChildren<Values = any> = (form: FormInstance<Values>) => React.ReactN
 type RcFieldProps<Values = any> = Omit<FieldProps<Values>, 'children'>;
 type ChildrenType<Values = any> = RenderChildren<Values> | React.ReactNode;
 
+export type HasFeedbackIcons = (itemStatus: {
+  status: ValidateStatus;
+  errors?: React.ReactNode[];
+  warnings?: React.ReactNode[];
+}) => { [key in ValidateStatus]?: React.ReactNode | boolean };
```

Comment:
```suggestion
}) => { [key in ValidateStatus]?: React.ReactNode };
```

React.ReactNode contains boolean.

Reply:
Yes, It has. Because you may want to hide some icons without other icons causing margins. I think it's useful.

Reply:
you're right. I'll remove `boolean`.